boards/sparkfun/micromod/micromod_nrf52840.dts

```diff
@@ -0,0 +1,197 @@
+	micromod_header: connector {
+		compatible = "sparkfun,micromod-gpio";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 4 0>,	/* A0 */
+			   <1 0 &gpio0 5 0>,	/* A1 */
+			   <2 0 &gpio0 27 0>,	/* D0 */
+			   <3 0 &gpio1 8 0>,	/* D1/CAM_TRIG */
+			   <4 0 &gpio0 15 0>,	/* I2C_INT# */
+			   <5 0 &gpio0 29 0>,	/* G0/BUS0 */
+			   <6 0 &gpio0 3 0>,	/* G1/BUS1 */
+			   <7 0 &gpio1 13 0>,	/* G2/BUS2 */
+			   <8 0 &gpio1 12 0>,	/* G3/BUS3 */
+			   <9 0 &gpio1 11 0>,	/* G4/BUS4 */
+			   <10 0 &gpio0 17 0>,	/* G5/BUS5 */
+			   <11 0 &gpio1 6 0>,	/* G6/BUS6 */
+			   <12 0 &gpio1 4 0>,	/* G7/BUS7 */
+			   <13 0 &gpio1 14 0>,	/* G8 */
+			   <14 0 &gpio0 9 0>,	/* G9/ADC_D-/CAM_HSYNC */
+			   <15 0 &gpio0 10 0>,	/* G10/ADC_D+/CAM_VSYNC */
+			   //<16 NO CONNECTED	/* G11/SWO */
+			   <17 0 &gpio0 20 0>;	/* SPI_CS */
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uicr {
+	nfct-pins-as-gpios;
+	gpio-as-nreset;
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "okay";
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-1 = <&uart0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&uart1 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "okay";
+	pinctrl-0 = <&uart1_default>;
+	pinctrl-1 = <&uart1_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&i2c0 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-1 = <&i2c0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&i2c1 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-1 = <&i2c1_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&spi2 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
+	cs-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>;
+	pinctrl-0 = <&spi0_default>;
+	pinctrl-1 = <&spi0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&qspi {
+	status = "okay";
+	pinctrl-0 = <&qspi_default>;
+	pinctrl-1 = <&qspi_sleep>;
+	pinctrl-names = "default", "sleep";
+	w25q128jv: w25q128jv@0 {
+		status="okay";
+		compatible = "nordic,qspi-nor";
+		reg = <0>;
+		// Currently, it supports only pp mode
+		writeoc = "pp";
+		// Currently, it supports only fastread and read2o
+		readoc = "fastread";
+		sck-frequency = <DT_FREQ_M(8)>;
+		jedec-id = [ef 70 18];
+		size = <DT_SIZE_M(128)>; // 128 Mbits
+		has-dpd;
+		t-enter-dpd = <3000>;
+		t-exit-dpd = <3000>;
+	};
+};
+
+&ieee802154 {
+	status = "okay";
+};
+
+&flash0 {
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x0000C000>;
+		};
+		slot0_partition: partition@c000 {
+			label = "image-0";
+			reg = <0x0000C000 0x00076000>;
+		};
+
+		slot1_partition: partition@82000 {
+			label = "image-1";
+			reg = <0x00082000 0x00076000>;
+		};
```

Comment:
if using swap using move then these sizes are not optimal, if you build mcuboot with debug logging then it will show a message explaining (optional)

Reply:
I used to use scratch instead of moving, I have read about it and I fixed it increasing primary slot. Now, the mentioned debug log is not displaying. Thanks!